Remotion/Data/DomainObjects.UnitTests/ReflectionUtilityTest.cs

```diff
@@ -98,6 +98,9 @@ public void GetAssemblyPath_FromNonPersistentAssembly ()
     [Test]
     public void GetAssemblyPath_FromUncPath ()
     {
+      if (!OperatingSystem.IsWindows())
+        Assert.Ignore("This test only works on Windows.");
+
       var assemblyMock = new Mock<FakeAssembly>(MockBehavior.Strict);
       assemblyMock.Setup(_ => _.Location).Returns(@"\\server\share\directory\assembly.dll");
       Assert.That(() => ReflectionUtility.GetAssemblyDirectory(assemblyMock.Object), Is.EqualTo(@"\\server\share\directory"));
```

Comment:
While this test is already sort of exotic, why doesn't it work on Linux? (I haven't checked if UNC is available on Linux but it's network, sooo...)